extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/TaskStartTimeoutFault.java

```diff
@@ -25,25 +25,34 @@
 import org.apache.druid.msq.util.MultiStageQueryContext;
 
 import java.util.Objects;
+import java.util.concurrent.TimeUnit;
 
 @JsonTypeName(TaskStartTimeoutFault.CODE)
 public class TaskStartTimeoutFault extends BaseMSQFault
 {
   static final String CODE = "TaskStartTimeout";
 
   private final int numTasks;
+  private final long timeout;
```

Comment:
Nit: we might want to document the additional field  in the reference.md for this fault  .

Reply:
Good idea. I added it.